src/style.css

```diff
@@ -1 +1,108 @@
 /* Styles go here */
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+a {
+  text-decoration: none;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ol,
+ul {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.header-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgb(255, 255, 255);
+  padding: 10px 0;
+  width: 924px;
+  margin: 0 auto;
+}
+
+.page-logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 753px;
+
+}
+
+.nav-item_linc {
+  display: block;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: rgb(0, 0, 0);
+  position: relative;
+  padding: 15px 0;
+}
+
+.nav-item_linc:hover {
+  color: rgb(0, 172, 220);
+}
+
+.nav-item_linc::after {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 4px;
+  background: rgb(0, 172, 220);
+  position: absolute;
+  bottom: -5px;
+  opacity: 0;
+  transition: opacity 0.4s ease-in-out;
+}
+
+.nav-item_linc.is-active::after {
+  opacity: 1;
+}
```

Comment:
why setting `::after` to all the links, when you can apply it only to `.is-active` one? 😁